packages/forms-web-app/src/controllers/examination/session/examination-session.js

```diff
@@ -1,9 +1,46 @@
+const logger = require('../../../lib/logger');
+
 const getExaminationSession = (session) => {
 	const examinationSession = session?.examination;
 	if (!examinationSession) throw new Error('No examination session');
 	return examinationSession;
 };
 
+const setExaminationUploadingState = (session, state) => {
+	const examinationSession = getExaminationSession(session);
+	if (typeof state !== 'boolean') throw new Error('Examination upload state is not a boolean');
+	examinationSession.uploading = state;
+	// Save is necessary to save the session in between states- https://stackoverflow.com/a/26532987
```

Comment:
comment feels necessary.

Reply:
I would leave it as we will have forgotten in 2 weeks time